scripts/generate_test_suites.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+
+parts = sys.argv[1]
+
+instrumentation_tests_path = 'instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests'
+core_tests = os.listdir(f'{instrumentation_tests_path}/core')
```

Comment:
Is this recursive? We might add "subpackages".

Reply:
Now it is recursive only for ui package, should we make recursive for core package also?

Reply:
I think so, otherwise adding a "subpackage" into core package won't be trivial.